tools/generators/epic-generator/index.spec.ts

```diff
@@ -45,7 +45,7 @@ function setupTest(packages: Package[]) {
 
   // response to 'gh auth'
   spawnSyncMock.mockReturnValueOnce({
-    output: ['Logged in to github.com'],
+    output: [['Logged in to github.com']],
```

Comment:
Can you provide some details on this change like you have for the others?

Reply:
Type 'string' is not assignable to type 'string[] | null'.ts(2322)

node 10
<img width="336" alt="image" src="https://user-images.githubusercontent.com/1223799/200523008-0d74db5e-b997-4c30-bea4-dc9c3d9f367d.png">

node 14
<img width="316" alt="image" src="https://user-images.githubusercontent.com/1223799/200522787-b26ac63c-d30e-4d3c-ae8a-b759a0534dbf.png">